include/net/ethernet.h

```diff
@@ -486,10 +486,10 @@ static inline
 enum ethernet_hw_caps net_eth_get_hw_capabilities(struct net_if *iface)
```

Comment:
Looks good, just fix the typo in commit message s/convertions/conversions/

Reply:
done